### PR TITLE
#156882157 min of 3 players, max player of 12

### PR DESCRIPTION
**Pull request by kenoseni**
#### What does this PR do?
Add a feature that allows minimum of 3 players and maximum of 12 and also invite other player by email
#### Description of Task to be completed?
Players can not start a game without a minimum of 3 players and other players cannot join when a maximum of 12 players have been reached. Players can also invite other players via email.
#### How should this be manually tested?
Clone the repo and cd into it. Run *npm start* and view the app in chrome, signup and test the features
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
156882157
#### Screenshots (if appropriate)
None
#### Questions:
None

### CODE REVIEW

**Comment by chuzksy-codeactive** [comment]
Good job, everything seems ok.

File: public/js/controllers/game.js [hunk 1]
```diff
@@ -1,184 +1,259 @@
...
+      // $scope.error = game;
+      $scope.userInfo = {};
+      $scope.messageInvite = '';
+
+      $scope.resetForm = () => {
+        $scope.user = {};
+      };
+
+      // Make request to sever if email is valid
+      $scope.getUserInfo = (email, searchForm) => {
```
Comment (andela-moe) [inline public/js/controllers/game.js:28]: In your view, I see that the parameters you pass to this method `getUserInfo(user, searchForm)` is that of the user, why then do you have this method expecting an **email** value? Is this a typo error or is there something I am missing here? 

Please address this.

**Review by andela-moe — COMMENTED** [review]
Please review comment

**Review by Spectrumsun — COMMENTED** [review]
Good job. Next time be more consistent with your variable name.

**Pull request closed without merging**